js/data/scripts/main.js

```diff
@@ -0,0 +1,95 @@
+"use strict";
+
+var onlyFr = false;
+
+
+
+document.addEventListener('DOMContentLoaded', function(){
+
+fetch('https://rawgit.com/DavidBruant/contenu-formations-web/master/js/data/tweets.json')
+.then(function(resp){return resp.json()})
+.then(function(tweets){
+	// var el = document.querySelector('#tweets');
+	// var tweets = JSON.parse(el.textContent)
+
+	// pour le premier tweet
+	// l'afficher
+	console.log(tweets[0]);
+	// afficher son message dans la console
+	console.log("Message : " + tweets[0].text);
+	// créer une div qui contient le texte du commit et ajouter la div au <body>
+	var divMessage = document.createElement('div');
+	document.body.appendChild(divMessage);
+
+	var el = document.querySelector('div');
+
+	tweets.forEach(function(tweet) {
+	    createTweetOl(tweet);
+	});
+
+	/******************************************************************
+	********************** SWITCH TWEETS DISPLAY **********************
+	******************************************************************/
+
+	document.getElementById('buttonTweets').addEventListener("click", function() {
+	    onlyFr = !onlyFr;
+	    var liFr = document.querySelectorAll(".en");
```

Comment:
I'll take a longer time on this in class, but you shouldn't have to read information from the DOM. Instead, you should be re-creating new elements from the data (and only read the information to generate the elements from the data).